res/Sectors/M1105/Phlask.txt

```diff
@@ -30,446 +30,445 @@
+2320                      C536623-4 Ni                           { -2 } (550-1) [8437] - -  - 324 18 Hv M4 V
+2412                      E659562-8 Ni Cy O:2516                 { -3 } (443+0) [8249] - -  - 110  8 Hv M9 V M9 V
+2414                      C639204-8 Lo                           { -2 } (D11+2) [1177] - -  - 123 13 Hv G5 V M9 V
+2415                      C654561-8 Ag Ni Cy O:2516              { -1 } (A41+1) [1419] - -  - 903 13 Hv M0 V
+2418                      A8A17S5-C Fl He                        {  2 } (C6A+1) [A99C] - K  - 704 10 Hv M2 V
+2419                      B4108S1-A Na Pi Ph                     {  2 } (A7A-1) [BA7D] - -  - 920 14 Hv A3 V
+2511                      D646562-7 Ag Ni Cy O:2516              { -2 } (644+3) [3317] - M  - 924 15 Hv G7 V
+2512                      B300201-E Lo Va                        {  1 } (D11-1) [437E] - K  - 214 14 Hv F0 V
+2516                      A4507S6-C De Po                        {  2 } (F68-2) [7947] - K  - 214 15 Hv G2 V
+2611                      D5638T5-3 Ri Ph                        { -1 } (A73+1) [77A5] - M  - 100 11 Hv M2 V M4 V
+2612                      D6677S6-2 Ag Ri Ga                     {  0 } (96B+0) [3772] - M  - 804 11 Hv M9 V
+2614                      X100623-0 Na Ni Va Dooa9 Fo            { -3 } (853+0) [4333] - -  R 713 12 Hv M9 V
+2616                      A543411-8 Ni Po                        { -1 } (533-4) [1338] - -  - 100 10 Hv M1 V M6 V
+2619                      A8A4002-D Fl Di                        {  1 } (E03+0) [0000] - K  - 004 16 Hv G7 V M9 V
+2713                      E5358S4-5 Ph                           { -2 } (A74-1) [665A] - -  - 414 18 Hv M5 V M2 V
+2714                      C100304-8 Lo Va AoaoW Pz               { -2 } (721+0) [2178] - -  A 604 10 Hv M2 V K5 V
+2716 Ao                   C666763-8 Ag Ri Ga (Aoao) Cy O:2619 Da {  1 } (A69+3) [485C] - -  A 101 12 Hv M3 V K0 V
+2718                      C679101-7 Lo AoaoW Pz                  { -2 } (401+4) [2194] - -  A 200 10 Hv M0 V
+2811                      D591523-2 Ni He                        { -3 } (A40+0) [4221] - M  - 203 12 Hv G9 V M7 V
+2817                      C876521-9 Ag Ni                        {  0 } (B45-3) [5568] - -  - 414 14 Hv M8 V
+2818                      C9B4105-7 Fl Lo AoaoW Pz               { -2 } (701+0) [1196] - -  A 314 13 Hv M4 V M9 V
+2913                      D77A202-7 Lo Wa                        { -3 } (A11+2) [11A2] - M  - 300  7 Hv F5 V M5 V
+2914                      D697561-7 Ag Ni Cy O:3116              { -2 } (C40-4) [2377] - M  - 402 12 Hv M2 V K7 V
+3011                      D455362-7 Lo O:3116                    { -3 } (C21+5) [111C] - -  - 312 12 Hv G6 V
+3012                      D5487S4-4 Ag Pi                        { -1 } (862+3) [9634] - M  - 102 14 Hv M1 V K9 V
+3016                      C4916S2-8 Ni He                        { -2 } (C50-4) [1456
```

Comment:
Per the T5 Second Survey rules this TL should be reduced to F or E. As a pop-A world I recommend TL-E. This will have a huge impact on the impact of the economy of the sector

Reply:
Can't see what needs to change in Phlask. Can you clarify

Reply:
I think this comment is in the wrong place. Sorry about that.

Reply:
Ok.